portugues/colaborando.md

```diff
@@ -37,8 +37,6 @@ A sua linha telefônica está uma droga hoje e você tem que baixar a ultima ver
 
 O Git te parece ser util, mas como você ainda não vive no século XXI, você está mais preocupado com esse tal de Windows.
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.